modules/c8000v/main.tf

```diff
@@ -0,0 +1,66 @@
+data "equinix_network_device_type" "this" {
+  category = "ROUTER"
+  vendor   = "Cisco"
+}
+
+data "equinix_network_device_platform" "this" {
+  device_type = data.equinix_network_device_type.this.code
+  flavor      = var.platform
+}
+
+data "equinix_network_device_software" "this" {
+  device_type = data.equinix_network_device_type.this.code
+  packages    = [var.software_package]
+  stable      = true
+  most_recent = true
+}
+
+resource "equinix_network_device" "non_cluster" {
+  lifecycle {
+    ignore_changes = [version, core_count]
```

Comment:
Any ideas why those two fields are ignored?

Reply:
removed ignore changes to allow vm resizing.